packages/addons-v5/package.json

```diff
@@ -50,7 +50,9 @@
     "reporter": "list",
     "recursive": true,
     "check-leaks": true,
-    "require": ["./test/init.js"],
+    "require": [
+      "./test/init.js"
+    ],
```

Comment:
This is from yarn. I think this is just updated linting. Nothing actually changing the value. Same for other linting updates made in this file.